std/algorithm.d

```diff
@@ -6626,9 +6626,9 @@ struct Until(alias pred, Range, Sentinel) if (isInputRange!Range)
     private bool predSatisfied()
     {
         static if (is(Sentinel == void))
-            return unaryFun!pred(_input.front);
+            return cast(bool) unaryFun!pred(_input.front);
         else
-            return startsWith!pred(_input, _sentinel);
+            return cast(bool) startsWith!pred(_input, _sentinel);
```

Comment:
I think the compiler sometimes has problems optimizing the ternary operator (I just remember reading about this ages ago, not sure if true anymore).

Anyway `?:` is just visual noise if `cast(bool)` works the same. :)